starter-code/src/assessment1.js

```diff
@@ -1,51 +1,68 @@
 // Write a function that returns the product of 2 numbers
 function product(x, y) {
-  //
+  return (x * y);
 }
 
 // Write a function that returns whether a given number is even
 function isEven(num) {
-  //
+  return num % 2 === 0;
 }
 
 // Return the largest of 2 numbers
 function maxOfTwoNumbers(a, b) {
-  //
+if (a > b) {
+  return a;
+} else {
+  return b;
+}
 }
 
 // Return the largest of 3 numbers
 function maxOfThreeNumbers(a, b, c) {
-  //
+
 }
 
 // Calculate the sum of an array of numbers
 function sumArray(numbers) {
-  //
+  
+  
+  let sum = 0;
+  
+
+  return sum;
 }
 
+
 // Return the largest number of a non-empty array
 function maxOfArray(numbers) {
-  //
+  
 }
 
 // Return the longest string in an array
 function longestString(strings) {
-  //
+ 
 }
 
 // Return whether a word is in an array
 function doesWordExist(wordsArr, word) {
-  //
+  const wordExists = wordsArr.includes(word);
+  return wordExists;
 }
 
 // Finding the first non-duplicate (non-repeating) word in an array
 function findUnique(wordsArr) {
-  //
+  
+  let newListOfWords = [];
+  for (const w of someArr) {
+    if (!newListOfWords.includes(w))
+      newListOfWords.push(w);
+  }
```

Comment:
this checks to see if the new array your creating contains the a word from the array and returns a new array. For this one you should check if the word is repeated in the original array and return the first word that does not repeat in it